.github/workflows/release.yml

```diff
@@ -40,6 +47,9 @@ jobs:
           GITHUB_TOKEN: '${{ secrets.GITHUB_TOKEN }}'
           NPM_TOKEN: '${{ secrets.NPM_TOKEN }}'
 
+      - name: Push to submodules
```

Comment:
Ditto here

Reply:
What are we pushing to submodules

Reply:
Maybe "push changesets to submodules"

Reply:
What would you recommend renaming this to to make it more clear?

Reply:
"push changesets to submodules"

Reply:
Done